app/src/main/java/com/keylesspalace/tusky/adapter/StatusBaseViewHolder.java

```diff
@@ -915,12 +915,18 @@ private void setupPoll(PollViewData poll, List<Emoji> emojis,
 
         if (expired || poll.getVoted()) {
             // no voting possible
-            pollAdapter.setup(poll.getOptions(), poll.getVotesCount(), poll.getVotersCount(), emojis, PollAdapter.RESULT);
+            View.OnClickListener viewThreadListener = v -> {
```

Comment:
There is already the same listener created in line 669 - it would be cool if we could reuse that somehow - how about just setting it on the whole RecyclerView?

Reply:
You mean on the view holder? Or I'm totally misunderstanding what you're getting at?

Reply:
I meant just doing `pollOptions.setOnClickListener(viewThreadListener);` in line 677, but it does not work I tried it out. Your solution is fine 👍

Reply:
Oh yeah, I looked at that too, but it apparently needs to be specifically on the result subview :shrug: